### PR TITLE
hate_speech_18 initial commit

### PR DESCRIPTION
**Pull request by czabo**

### CODE REVIEW

File: datasets/hate_speech_18/README.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids:
+- intent-classification
```
Comment (lhoestq) [inline datasets/hate_speech_18/README.md:18]:
```suggestion
---
annotations_creators:
- found
language_creators:
- found
languages:
- en
licenses:
- cc-by-sa-3.0
multilinguality:
- monolingual
size_categories:
- 10K<n<100K
source_datasets:
- original
task_categories:
- text-classification
task_ids:
- intent-classification
---
```

**Review by lhoestq — COMMENTED** [review]
Nice thanks !

Can you rename the dataset folder and the dataset script name `hate_speech18` instead of `hate_speech_18` to follow the snake case convention we're using ?

Also it looks like the dummy_data.zip file is quite big (almost 4MB).
Can you try to reduce its size ?

To do so feel free to take a look inside it and remove all the unnecessary files or chunks of texts. The idea is to only keep a few examples

**Comment by czabo** [comment]
> Nice thanks !
> 
> Can you rename the dataset folder and the dataset script name `hate_speech18` instead of `hate_speech_18` to follow the snake case convention we're using ?
> 
> Also it looks like the dummy_data.zip file is quite big (almost 4MB).
> Can you try to reduce its size ?
> 
> To do so feel free to take a look inside it and remove all the unnecessary files or chunks of texts. The idea is to only keep a few examples

Done, thanks!

**Comment by yjernite** [comment]
Re-opened in https://github.com/huggingface/datasets/pull/1486

**Pull request closed without merging**